File: java/src/test/java/com/thoughtworks/pairing/MyTest.java
package com.thoughtworks.pairing;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;


public class MyTest {

    @Test
    public void myFirstTest() {
        MyClass x = new MyClass();
        assertEquals(5, x.sum(2,3));
    }
}

File: kotlin/src/test/kotlin/com/thoughtworks/pairing/MyTest.kt
package com.thoughtworks.pairing

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test


class MyTest {

    @Test
    fun `my first test`() {
        val x = MyClass()
        assertEquals(5, x.sum(3, 2))
    }

}
